main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 2.49"
+    }
+  }
+
+  backend "azurerm" {
+      resource_group_name = "McLaren1_LachlanPearse_ProjectExercise"
+      storage_account_name = "lptodoappterraformstate"
+      container_name = "tfstate"
+      key = "terraform.tfstate"
+  }
+}
+provider "azurerm" {
+  features {}
+}
+data "azurerm_resource_group" "main" {
+  name = "McLaren1_LachlanPearse_ProjectExercise"
+}
+
+data "azurerm_cosmosdb_account" "main" {
+  name                = "lp-projectexercise-db"
+  resource_group_name = data.azurerm_resource_group.main.name
+
+}
+
+data "azurerm_cosmosdb_mongo_database" "main" {
+  name                = "todo_app"
+  resource_group_name = data.azurerm_resource_group.main.name
+  account_name        = data.azurerm_cosmosdb_account.main.name
```

Comment:
Note that the intention here was to add & manage a new database/account using the `resource` keyword as opposed to importing an existing one with `data` - but since everything is working I'm happy that there's no strong need to take that approach!